site/en/docs/extensions/mv3/intro/index.md

```diff
@@ -10,40 +10,26 @@ updated: 2022-09-28
 
```

Comment:
GitHub isn't letting me suggest in the metadata lines. Please change the `updated` date to today.

In the future:
* If mere polish the existing content, don't change `updated`.
* If you you rewrite the page, do change `updated'.

Reply:
Done. Thanks!